### DISCUSSION TITLE
希望可以支持win10 19045.5131和以前更早的版本，谢谢大佬

### DISCUSSION THREAD

**Discussion by xiaoyaozishiye** [xfangfang/wiliwili#442] [Ideas]
![2024-11-21](https://github.com/user-attachments/assets/e2ac5472-4a2e-42a3-a1e4-ff2647ce4a26) 我的win10无法安装这个

**Comment by xfangfang** [comment]
目前windows版默认使用D3D图形API，在性能上是和UWP同级别的水平，如果没有特殊需求，使用默认的Windows版即可。
